larreco/QuadVtx/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+
+art_make(BASENAME_ONLY
+         LIB_LIBRARIES
+           larcorealg_Geometry
+           ${ART_FRAMEWORK_SERVICES_REGISTRY}
```

Comment:
Only modules/services should use services.  The `${ART_FRAMEWORK_SERVICES_REGISTRY}` link line should be removed.  Also, as this library appears to only use facilities from the STL and ROOT, only `${ROOT_*}` libraries should be included in the `LIB_LIBRARIES` list.

Reply:
As you can probably tell, the makefile is completely cargo-culted. I'll happily make these changes, or just allow anyone actually knowledgeable to rewrite as part of the pull.